sdcm/nemesis.py

```diff
@@ -1088,6 +1089,70 @@ def disrupt_ldap_connection_toggle(self):
         ContainerManager.unpause_container(self.tester.localhost, 'ldap')
         self.log.info('finished with nemesis')
 
+    def disrupt_ldap_role_add_remove_permission(self):
+        """
+        Scenario:
+        1. Create new user in Scylla
+        2. Create new role in Ldap containing the new user as a member.
+        3. Verify this new user is unauthorised on resources (create a keyspace).
+        4. Create a new super-user role in Scylla.
+        5. Create a new super-user role in Ldap, with the new user as a member.
+        6. Verify this new user is authorised on resources.
+        7. Modify Ldap role to remove membership of new user.
+        8. Verify the new user is unauthorised on resources (create a table).
+        9. Do clean-up: delete Ldap Role, Scylla roles and keyspace/tables.
+
+        """
+        if not self.cluster.params.get('use_ldap_authorization'):
+            raise UnsupportedNemesis('Cluster is not configured to run with LDAP authorization, hence skipping')
+        if not self.cluster.params.get('ldap_server_type') == LdapServerType.MS_AD:
+            raise UnsupportedNemesis('Cluster is not configured to run with open LDAP authorization, hence skipping')
+        if not self.target_node.is_enterprise:
```

Comment:
this should be the 1st check, because even if someone mistakenly sets `use_ldap_authorization` on OSS, we should already skip it

Reply:
fixed.